routes/api-routes.js

```diff
@@ -51,7 +52,11 @@ module.exports = function (app) {
 
         if (source === "home") {
           // NEED TO REMOVE PLACES THAT ARE NOT IN OUR DATABASE
-          res.json(detailedPlaces);
+          // res.json(detailedPlaces);
+          // console.log(db.Bathroom);
+          db.Bathroom.findAll({}).then((dbBathrooms) => {
+            res.json({dbBathrooms, detailedPlaces});
```

Comment:
I was thinking this should happen on the server, the combining of detailed places from Google with the bathroom information we have on said locations, but we can still merge this in, since Matt is working on it as well.